examples/browser/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Browser Native node-wot tryout</title>
+    <!-- Foundation CSS framework -->
+    <link rel='stylesheet' href='https://cdn.jsdelivr.net/foundation/5.0.2/css/foundation.min.css'>
+
+    <!-- Font Awesome icons -->
+    <link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.0.3/css/font-awesome.css'>
+
+    <!-- defer to not block rendering -->
+    <script src="https://cdn.jsdelivr.net/npm/@json-editor/json-editor@latest/dist/jsoneditor.min.js" defer></script>
+    <script src="https://cdn.jsdelivr.net/npm/@node-wot/browser-bundle@latest/dist/wot-bundle.min.js" defer></script>
+    <script src="index.js" defer></script>
```

Comment:
When loading in FF66 I besides the failing resource I get some more warnings.

e.g., ReferenceError: JSONEditor is not defined

Maybe it would be good to add some try catch code to let the user know that somethings did not work as expected.

Reply:
The first defer should be removed. Thanks I will fix this.